include/d/actor/d_a_tori_flag.h

```diff
@@ -29,12 +47,7 @@ class daTori_Flag_c : public fopAc_ac_c {
     /* 0x4D0 */ dCcD_Cyl mCyl2;
 };
 
-class daTori_Flag_HIO_c {
-public:
-    daTori_Flag_HIO_c();
-
-public:
-    /* Place member variables here */
-};
+extern daTori_Flag_HIO_c l_HIO;
+extern cXyz l_flag_offset;
```

Comment:
The `l_` prefix means local, so these don't need to go in the header since they're local to this one TU and won't be included for use in other TUs